### PR TITLE
Fixes RC lost status for failsafe types of RX with SBus out

### PR DESCRIPTION
**Pull request by OpenUAS**
Fixes issue  #2306 . Tested with two different receivers with SBus out. More testing welcomed. Looking at the code it is likely no regression only imprevment. A flyaway also less likely with this fix, if AUTO2 set on RC loss. Else RC loss was available but not detected, steering not possible in manual/auto1
Some defines in radio_control.h re-definable for more timeout flexibility.

### CODE REVIEW

File: sw/airborne/subsystems/radio_control/sbus_common.c [hunk 1]
```diff
@@ -27,6 +27,7 @@
 #include "subsystems/radio_control/sbus_common.h"
 #include BOARD_CONFIG
 #include <string.h>
+#include <stdbool.h>
```
Comment (gautierhattenberger) [inline sw/airborne/subsystems/radio_control/sbus_common.c]: it shouldn't be necessary

File: sw/airborne/subsystems/radio_control/sbus.c [hunk 2]
```diff
@@ -72,16 +72,35 @@ static inline void sbus_decode_event(void)
...
-    radio_control.frame_cpt++;
-    radio_control.time_since_last_frame = 0;
-    if (radio_control.radio_ok_cpt > 0) {
-      radio_control.radio_ok_cpt--;
-    } else {
-      radio_control.status = RC_OK;
-      NormalizePpmIIR(sbus.pulses, radio_control);
-      _received_frame_handler();
+
+  if (sbus.frame_available)
```
Comment (gautierhattenberger) [inline sw/airborne/subsystems/radio_control/sbus.c]: please run fix_style script

Comment (OpenUAS) [reply sw/airborne/subsystems/radio_control/sbus.c]: will do...

File: sw/airborne/subsystems/radio_control/sbus_common.c (hunk 1 above)
Comment (OpenUAS) [reply sw/airborne/subsystems/radio_control/sbus_common.c]: Yes, included via elsewhere but makes executable not bigger and IDE happier with syntax highlight ;)

**Comment by OpenUAS** [comment]
Thx for the great response time really appreciated.

**Comment by OpenUAS** [comment]
Thx for the great response time, really appreciated. Now we can re-testfly various new things for PPRZ for OBC2018

**Comment by fvantienen** [comment]
Please do not merge your own pull requests.. the reviews are there to improve the code quality.

**Pull request merged**